package.json

```diff
@@ -2,23 +2,21 @@
   "name": "nodeapi",
   "version": "1.3.0",
   "description": "node api for COINS",
-  "main": "index.js",
+  "main": "dist/index.js",
```

Comment:
out of scope for this project, BUT, the because of directory structures, this as a distributed module still wont work if imported.  e.g. `../../../package.json` is now at a different depth if someone were to import nodeapi and try to run it.